pattern/crossfader.c

```diff
@@ -18,7 +18,7 @@ void crossfader_init(struct crossfader * crossfader) {
     // Render targets
     glGenFramebuffersEXT(1, &crossfader->fb);
     glGenTextures(1, &crossfader->tex_output);
-    if((e = glGetError()) != GL_NO_ERROR) FAIL("OpenGL error: %s\n", gluErrorString(e));
+    if((e = glGetError()) != GL_NO_ERROR) FAIL("OpenGL error: %s\n", GLU_ERROR_STRING(e));
```

Comment:
Not for this patch, but might want to macro-ize this so it can be stripped out in release; the overhead can be pretty significant in some cases.